### ISSUE TITLE
Draft API Report: Learning api

### ISSUE THREAD

**Issue by rottitime** [i-dot-ai/ellandi#716]

**Comment by rottitime** [comment]
@nmenezes0  did you want to catch up about restructuring for consistency? I made this simpler as the page does not require any 'DataGrid', sorting or pagination. It is all mostly content on page

![image](https://user-images.githubusercontent.com/1698584/200516622-85ccdf02-1c50-4c64-9c57-8d858092d422.png)

**Comment by nmenezes0** [comment]
> @nmenezes0 did you want to catch up about restructuring for consistency? I made this simpler as the page does not require any 'DataGrid', sorting or pagination. It is all mostly content on page
> 
> ![image](https://user-images.githubusercontent.com/1698584/200516622-85ccdf02-1c50-4c64-9c57-8d858092d422.png)

I think that this is probably fine for me to get started and I'll shout when I have questions. For users I was going to set to `all`, `line_managers` and `mentors` (as opposed to capitalised as above) - to be consistent with the other endpoint? Is that ok? (Happy to do either though.)

**Comment by rottitime** [comment]
> I think that this is probably fine for me to get started and I'll shout when I have questions. For users I was going to set to `all`, `line_managers` and `mentors` (as opposed to capitalised as above) - to be consistent with the other endpoint? Is that ok? (Happy to do either though.)

Yes that's fine for `all`, `line_managers` and `mentors`. thanks
